x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/user/RemoteMonitoringUser.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+package org.elasticsearch.xpack.core.security.user;
+
+import org.elasticsearch.Version;
+import org.elasticsearch.xpack.core.security.support.MetadataUtils;
+
+/**
+ * Built in user for remote monitoring: collection as well as indexing.
+ */
+public class RemoteMonitoringUser extends User {
+
+    public static final String NAME = UsernamesField.REMOTE_MONITORING_NAME;
+    public static final String COLLECTION_ROLE_NAME = UsernamesField.REMOTE_MONITORING_COLLECTION_ROLE;
+    public static final String INDEXING_ROLE_NAME = UsernamesField.REMOTE_MONITORING_INDEXING_ROLE;
+
+    public static final Version DEFINED_SINCE = Version.V_6_5_0;
+    public static final BuiltinUserInfo USER_INFO = new BuiltinUserInfo(NAME, COLLECTION_ROLE_NAME+","+INDEXING_ROLE_NAME, DEFINED_SINCE);
```

Comment:
@tvernum @albertzaharovits I could use some guidance here. I'm not sure I'm doing the right thing with the 2nd argument to the `BuiltinUserInfo` constructor here. Thoughts/suggestions?

Reply:
Do you even need this `BuiltinUserInfo` field?
As far as I can see, that class is never used. I'll need to verify the git history, but my recollection is that we removed all need for some time ago.

Reply:
Ha, good catch! Looking through the code, I don't see the `USER_INFO` constant being referenced anywhere, so I'm going to remove it now. Thanks!

Reply:
It looks like it's been redundant since 5.6 (see fe9d99d), but has never been removed and keeps being copied into new user objects.

Reply:
Related: https://github.com/elastic/elasticsearch/pull/34541